playbooks/roles/install_lbaasv2_driver/tasks/main.yml

```diff
@@ -21,6 +21,11 @@
     state: present
   when: not use_pip|bool
 
+- name: Install git
+  package:
+    name:  '{{ versioned_git_yum_package }}'
```

Comment:
You can make this variable ambiguous, trusting that ansible will do the right thing when it installs git on a debian system (or anything else it supports).

'{{ versioned_git_package }}'